addrcmd.go

```diff
@@ -0,0 +1,76 @@
+package main
+
+import (
+	"fmt"
+
+	"github.com/urfave/cli/v2"
+	"golang.org/x/xerrors"
+
+	"github.com/filecoin-project/go-address"
+	"github.com/filecoin-project/go-state-types/abi"
+	"github.com/filecoin-project/lotus/chain/types/ethtypes"
+	"github.com/ipfs/go-cid"
+)
+
+func filToEthAddr(cctx *cli.Context) error {
+	ctx := cctx.Context
+	args := cctx.Args()
+	actorAddrString := args.Get(0)
+	addr, err := address.NewFromString(actorAddrString)
+	if err != nil {
+		return err
+	}
+
+	switch p := addr.Protocol(); p {
+	case address.ID, address.Delegated:
+		eaddr, err := ethtypes.EthAddressFromFilecoinAddress(addr)
+		if err != nil {
+			return err
+		}
+		fmt.Println(eaddr.String())
+	case address.Actor, address.BLS:
+	default:
+		return fmt.Errorf("error unsupported address type %v", p)
+	}
+
+	bstore, tsk, err := getState(cctx)
+	if err != nil {
+		return err
+	}
+
+	cbs := &countingBlockstore{Blockstore: bstore, m: make(map[cid.Cid]int)}
+	ebs := NewEphemeralBlockstore(cbs)
+	sm, err := newFilStateReader(ebs)
+	if err != nil {
+		return xerrors.Errorf("unable to initialize a StateManager: %w", err)
+	}
+
+	ts, err := sm.ChainStore().GetTipSetFromKey(ctx, tsk)
+	if err != nil {
+		return xerrors.Errorf("unable to load target tipset: %w", err)
+	}
+
+	stateTree, err := sm.StateTree(ts.ParentState())
+	if err != nil {
+		return err
+	}
+	idAddr, err := stateTree.LookupID(addr)
```

Comment:
Not sure if it's worth trying to make the errors here prettier, for example telling people that for say an f1 address they need to send some FIL to it. Could also potentially add to the command readme even though it's more about Filecoin than the command itself.